src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const types of actions) {
```

Comment:
[Checklist](https://github.com/mate-academy/js_stateful-object/blob/master/checklist.md) 3rd item, you're iterating through array of `action`, not `types`